src/ui/component/field/field.tsx

```diff
@@ -0,0 +1,108 @@
+import { type FC, type ChangeEvent, useCallback, useState, useRef } from 'react'
+import classNames from 'classnames'
+import { Icon } from '@/ui/component/icon/icon'
+import './field.scss'
+
+type FieldProps = {
+  id: string
+  label: string
+  onChange: (value: string) => void
+  value?: string
```

Comment:
If `type = 'number'`, value's type should be `number`, no?

Reply:
HTML input value are always strings, even this may seem unintuitive for input `number` type 😅

Reply:
🤪